README.md

```diff
@@ -220,7 +234,7 @@ For more advanced usage or to programmatically trigger the workflow from scripts
 
 ### Upload a Worker Version
 
-To create a new version of your Worker that is not deployed immediately, use the `wrangler versions upload --experimental-versions` command. Worker versions created in this way can then be deployed all at once at a later time or gradually deployed using the `wranger versions deploy --experimental-versions` command or via the Cloudflare dashboard under the Deployments tab. For now, the `--experimental-versions` flag and wrangler v3.40.0 or above is required to use this feature.
```

Comment:
I would confirm with wrangler team to see if `--experimental-versions` flag is still required or not, looks like it's still referenced here for [example](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/CONTRIBUTING.md#changesets)

Reply:
It has been confirmed!